applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_3D_interface_law.h

```diff
@@ -121,6 +121,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) LinearElastic3DInterfaceLaw : public
      * @param rValue output: the value of the specified variable
      */
     bool& GetValue(const Variable<bool>& rThisVariable, bool& rValue) override;
+    using LinearElastic2DInterfaceLaw::GetValue;
```

Comment:
Looks like this is a candidate for using a plane strain and a 3D stress state.

Reply:
In fact, we may want to look into the differences with the recently introduced class `GeoIncrementalLinearElasticInterfaceLaw`, and discuss whether we'd still like to keep this class or replace it by the new one.

Reply:
You are right. These should disappear when we remove the interface elements that derive from planes i.s.o. 2 lines and volumes i.s.o. 2 planes. They use continuum material properties ( E, nu ) i.s.o. distributed stiffnesses that are natural for interfaces.